radiomics/featureextractor.py

```diff
@@ -284,6 +284,9 @@ def enableFeatureClassByName(self, featureClass, enabled=True):
     """
     Enable or disable all features in given class.
     """
+    if not featureClass in self.getFeatureClassNames():
```

Comment:
@fedorov, flake 8 fails due to this line: if you compare with "not" and "in", flake 8 requires you to put these together, i.e. `if featureClass not in self.getFeatureClassNames():`